gql_code_builder/lib/src/operation/data.dart

```diff
@@ -223,7 +223,9 @@ Method _buildGetter(
                 .code
             : node.selectionSet == null
                 ? fieldTypeDef == null
-                    ? dataField.asA(returns).code
+                    ? returns.symbol == "double"
```

Comment:
Ideally, there should be a generalized coercer functionality that let's this problem be solved for custom types as well.

But for now this is good enough.